### DISCUSSION TITLE
[Projects Beta] Retain the repository navigation when viewing a project

### DISCUSSION THREAD

**Discussion by stumorgan** [community/community#16527] [Projects and Issues]
When I navigate into an organization project from the Projects tab within a repository, I'd like it to still show the navigation bar for that repository up top so I can quickly switch to Issues, etc.  Currently when you navigate into the organization project you can only see that project with no navigation to anything else possible.

**Comment by bennypowers** [comment]
Alternatively, even a link to the project's organization in the page header would be helpful. 

As it is now, the workflow to get back to the repo from the org project is:

1. click a random issue to open it in sidebar
2. click open in new tab
3. navigate from there

OR if the repo is in your frequents,

1. go to github homepage
2. navigate to the repo from recent repos

🙈 very awkward

**Reply by markcaron** [reply]
Agreed. At the very least, keeping a link to the project's organization in the page header is essential. Info architecture-wise, it makes sense as well, given projects are directly under orgs.

So incredibly awkward without this.

**Reply by markcaron** [reply]
<img width="856" alt="image" src="https://user-images.githubusercontent.com/1890950/170036951-115d6c14-69f6-4988-892f-f64dc5b76d8c.png">
